.devcontainer/init.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Install and use nvm version from .nvmrc
+source $NVM_DIR/nvm.sh
+nvm install
+nvm use
```

Comment:
I just rebuilt my codespace and it seems that nvm is already install and using v19 as default. While v19 is stable it is not LTS.  which version does the above use?

lts/* seems to give me the latest LTS of v18

Reply:
Yeah I always have to tell it to use LTS and then alias the default to be 18

Reply:
Yeah, nvm is installed already -- just not using LTS. This step just installs the LTS specified in the `.nvmrc` and sets it to the default. I should clarify the comment.

Currently we have `lts/gallium` (corresponding to Node 16) in the `.nvmrc` -- should we update that or otherwise just use `lts/*` here?

Reply:
v16 is good to me.